spec/suites/layer/PopupSpec.js

```diff
@@ -229,6 +229,18 @@ describe('Popup', function () {
 		L.Icon.Default.prototype.options.popupAnchor = popupAnchorBefore;
 	});
 
+	it("prevents an underlying map click for Layer", function () {
```

Comment:
I started out with the intent to check for a popup, but that is one symptom of an underlying issue: the click is getting through to the map. So maybe check for both? The tests would be:

1. Make sure popups work on Paths
2. Make sure Paths with popups don't propagate clicks to map (what I added)
3. Make sure that when the Path and map both have a popup, the popup matches the path._popup (to add)

Sound good?